ulib/gmake/fstar.mk

```diff
@@ -1,4 +1,7 @@
 HINTS_ENABLED?=--use_hints --use_hint_hashes
+PROTECT_TOP_LEVEL_AXIOMS?=--protect_top_level_axioms true
+WARN_ERROR=
+OTHERFLAGS+=$(PROTECT_TOP_LEVEL_AXIOMS) $(WARN_ERROR) --z3cliopt 'timeout=600000'
```

Comment:
Note, I am setting a 10 minute hard timeout on individual Z3 queries. WIthout it, I find that Z3 can run forever on the CI machine and we get no feedback about a build. But, we could revert this ...